README.md

```diff
@@ -2,6 +2,30 @@
 
 Precompile-EVM is a repository for registering precompiles to Subnet-EVM without forking the Subnet-EVM codebase. Subnet-EVM supports registering external precompiles through `precompile/modules` package. By importing Subnet-EVM as a library, you can register your own precompiles to Subnet-EVM and build it together with Subnet-EVM.
 
+## Environment Setup
+
+To effectively build, run and test a Precompile-EVM, you need software such as golang, node,[AvalancheGo](https://github.com/ava-labs/avalanchego), and [Avalanche Network Runner](https://github.com/ava-labs/avalanche-network-runner). 
```

Comment:
I think we should be more specific here rather than being vague regarding the dependencies. An alternative would be the following:

"To effectively build, run, and test Precompile-EVM, the following is a (non-exhaustive) list of dependencies that you will need:
- Golang
- Node.js
- [AvalancheGo](https://github.com/ava-labs/avalanchego)
- [Avalanche Network Runner](https://github.com/ava-labs/avalanche-network-runner)
"